frontend/src/Society/Society.tsx

```diff
@@ -0,0 +1,27 @@
+import classes from './Society.module.css';
+
+type SocietyProps = {
+    name: string;
+    image: string;
+    //These parameters can be passed as props otherwise some fault value maybe(?)
+    backgroundPositionY?: string;
+    color?:string
+};
+
+function Society(props: SocietyProps) {
+  return (
+    <div className={classes.container}>
+        <div
+        className={classes.picture}
+        style={{
+        backgroundImage: `url(${props.image})`,
+        backgroundPositionY: props.backgroundPositionY,
+        }}/>
+        <div className={classes.socNameDiv} style={{backgroundColor: props.color}}>
```

Comment:
update corresponding class names here
also note if `props.color` is undefined the background colour will also be undefined. i'd recommend doing something like this:
`style={{ backgroundColor: props.color ? props.color : 'hsl(some, default, value)' }}`